Projects/UOContent/Commands/Generic/Implementors/ObjectConditional.cs

```diff
@@ -22,9 +21,9 @@ public ObjectConditional(Type objectType, ICondition[][] conditions)
 
         public Type Type { get; }
 
-        public bool IsItem => Type == null || Type == typeofItem || Type.IsSubclassOf(typeofItem);
+        public bool IsItem => Type == null || Type == OfItem || Type.IsSubclassOf(OfItem);
```

Comment:
I think it might be easier to read it if we used:
```cs
Type.IsAssignableTo(OfItem);
```

Thoughts?

Reply:
I think `OfItem.IsAssignableFrom(t)` would be marginally faster for some cases than checking for subclass. Personally I have no preference as far as readability is concerned.

Reply:
Dropped type equality and replaced with `IsAssignableTo`  in 10d50a9